pyiron_contrib/atomistic/interactive/mixer.py

```diff
@@ -0,0 +1,323 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+import numpy as np
+from pyiron_base import JobStatus, GenericJob, GenericParameters
+from pyiron.atomistics.job.interactivewrapper import InteractiveWrapper, ReferenceJobOutput
+
+__author__ = "Jan Janssen"
+__copyright__ = "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - " \
+                "Computational Materials Design (CM) Department"
+__version__ = "1.0"
+__maintainer__ = "Jan Janssen"
+__email__ = "janssen@mpie.de"
+__status__ = "development"
+__date__ = "Sep 1, 2018"
+
+
+class Mixer(InteractiveWrapper):
+    """
+    Mixing forces of two quantum engines
```

Comment:
I think I would prefer "interpreters" to "engines", and ", e.g. Vasp." might be helpful to add.

Reply:
Actually, when I looked through the code I didn't see anything that's quantum-specific. If this is correct, it should be "atomistic interpreters, e.g. a Lammps or Vasp job" instead of "quantum engines".